rpi/controller.py

```diff
@@ -157,15 +157,15 @@ def get_tag(self):
 # the get_by_tag method will return the member info corresponding to a keyfob
 class Members(object):
     def __init__(self, filename=membership_file):
-        f = open(filename, 'rb')
+        f = open(filename, 'r')
         self.members = [i for i in csv.DictReader(f, delimiter=',')]
 
     def get_by_tag(self, tag_id):
         for m in self.members:
-            #print "Checking tag ", tag_id, " against member ", m['Name']
-            #print "RFID: ", m['RFID']
-            #print "format_id: ", format_id(m['RFID'])
-            #print "wiegandify: ", wiegandify(format_id(m['RFID']))
+            # print("Checking tag ", tag_id, " against member ", m['Name'])
+            # print("RFID: ", m['RFID'])
+            # print("format_id: ", format_id(m['RFID']))
+            # print("wiegandify: ", wiegandify(format_id(m['RFID'])))
```

Comment:
I'd nearly pull these out if we're not using them.
Maybe move them to a debug function that only gets called if a verbose debug flag is set?

Reply:
currently, we always log with verbose debug set to true. https://github.com/site-3/doorbot-firmware/blob/9d0ca482e70cf12217285ce91a6ac135c92f37c9/rpi/controller.py#L25 My next project will be to update how we manage logs, and with that, I would like to change the verbose logging, and then this is helpful again. It could very well be removed, and if needed, grabbed from the git history, but it's for my direct next task, so it won't be here long.